packages.windows/dcos-image-deps/buildinfo.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
This is a fine example of a buildinfo that I don't want to have to maintain copies of.